zebra-consensus/src/checkpoint/list/tests.rs

```diff
@@ -274,14 +277,21 @@ fn checkpoint_list_hard_coded_max_gap_testnet() -> Result<(), BoxError> {
     checkpoint_list_hard_coded_max_gap(Testnet)
 }
 
-/// Check that the hard-coded checkpoints are within `MAX_CHECKPOINT_HEIGHT_GAP`.
+/// Check that the hard-coded checkpoints are within [`MAX_CHECKPOINT_HEIGHT_GAP`],
+/// and a calculated minimum number of blocks. This also checks the heights are in order.
 ///
-/// We can't test the block byte limit, because we don't have access to the entire
-/// blockchain in the tests. But that's ok, because the byte limit only impacts
-/// performance.
+/// We can't test [`MAX_CHECKPOINT_BYTE_COUNT`] directly, because we don't have access to the
+/// entire blockchain in the tests. Instead, we check the number of maximum-size blocks in a
```

Comment:
###### Optional

```suggestion
/// We can't test [`MAX_CHECKPOINT_BYTE_COUNT`] directly, because we don't have access to a large
/// enough blockchain in the tests. Instead, we check the number of maximum-size blocks in a
```